docs/blog/microbit/arcadeshield-ext-sim.md

```diff
@@ -0,0 +1,37 @@
+# Arcade shield extension/simulator for the micro:bit (V2)
+
+**Posted on January 16, 2025 by [Thomas Ball](https://github.com/thomasjball)**
+
+The micro:bit V2 has an amazing amount of computational power for a tiny device. In fact, it’s powerful enough to drive a small color display. If you have ever used [MakeCode Arcade](https://arcade.makecode.com/), you may have noticed there are a variety of [Arcade shields](https://makecode.com/blog/microbit/arcade-shield) (also called expansion boards) now commercially available. The micro:bit V2 plugs into the shield, making the combination a retro gaming device:
```

Comment:
We do have a more recent blog post about the micro:bit Arcade shields you could reference - https://makecode.com/blog/arcade/arcade-on-microbit-xbox